base/unicode/properties.jl

```diff
@@ -0,0 +1,168 @@
+     (:Lt, 3,  :TitleCase,               :Upper,       "Letter, titlecase"),
+     (:Lm, 4,  :ModifierLetter,          :Letter,      "Letter, modifier"),
+     (:Lo, 5,  :OtherLetter,             :Letter,      "Letter, other"),
+     (:Mn, 6,  :NonSpacingMark,          :Mark,        "Mark, nonspacing"),
+     (:Mc, 7,  :CombiningMark,           :Mark,        "Mark, spacing combining"),
+     (:Me, 8,  :EnclosingMark,           :Mark,        "Mark, enclosing"),
+     (:Nd, 9,  :DecimalDigit,            :Number,      "Number, decimal digit"),
+     (:Nl, 10, :NumericLetter,           :Number,      "Number, letter"),
+     (:No, 11, :OtherNumber,             :Number,      "Number, other"),
+     (:Pc, 12, :ConnectorPunctuation,    :Punctuation, "Punctuation, connector"),
+     (:Pd, 13, :DashPunctuation,         :Punctuation, "Punctuation, dash"),
+     (:Ps, 14, :OpenPunctuation,         :Punctuation, "Punctuation, open"),
+     (:Pe, 15, :ClosePunctuation,        :Punctuation, "Punctuation, close"),
+     (:Pi, 16, :InitialQuotePunctuation, :Punctuation, "Punctuation, initial quote"),
+     (:Pf, 17, :FinalQuotePunctuation,   :Punctuation, "Punctuation, final quote"),
+     (:Po, 18, :OtherPunctuation,        :Punctuation, "Punctuation, other"),
+     (:Sm, 19, :MathSymbol,              :Symbol,      "Symbol, math"),
+     (:Sc, 20, :CurrencySymbol,          :Symbol,      "Symbol, currency"),
+     (:Sk, 21, :ModifierSymbol,          :Symbol,      "Symbol, modifier"),
+     (:So, 22, :OtherSymbol,             :Symbol,      "Symbol, other"),
+     (:Zs, 23, :SpaceSeparator,          :Separator,   "Separator, space"),
+     (:Zl, 24, :LineSeparator,           :Separator,   "Separator, line"),
+     (:Zp, 25, :ParagraphSeparator,      :Separator,   "Separator, paragraph"),
+     (:Cc, 26, :ControlChar,             :Other,       "Other, control"),
+     (:Cf, 27, :FormatChar,              :Other,       "Other, format"),
+     (:Cs, 28, :SurrogateChar,           :Other,       "Other, surrogate"),
+     (:Co, 29, :PrivateUseChar,          :Other,       "Other, private use"))
+    @eval const global $nam = $(Code(val))
+    @eval abstract $cat <: $typ
+    @eval Base.convert(::Type{Code}, ct::$cat) = $(Code(val))
+    @eval @doc $(string("Unicode Category Code: ",des)) $nam
+    @eval @doc $(string("Unicode Category Type: ",des)) $cat
+end
+
+const c2t = [NotAssignedChar, UpperCase, LowerCase, TitleCase, ModifierLetter, OtherLetter,
+             NonSpacingMark, CombiningMark, EnclosingMark,
+             DecimalDigit, NumericLetter, OtherNumber,
+             ConnectorPunctuation, DashPunctuation, OpenPunctuation, ClosePunctuation,
+             InitialQuotePunctuation, FinalQuotePunctuation, OtherPunctuation,
+             MathSymbol, CurrencySymbol, ModifierSymbol, OtherSymbol,
+             SpaceSeparator, LineSeparator, ParagraphSeparator,
+	     ControlChar, FormatChar, SurrogateChar, PrivateUseChar]
```

Comment:
Grrr!  I recently moved from Epsilon (an Emacs-like editor) to Emacs, because of the Julia support in Emacs, but there are still some things that I haven't adjusted to yet.